riotfile.py

```diff
@@ -795,6 +800,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "django-q": latest,
             },
             env={
+                "DD_CIVISIBILITY_ITR_ENABLED": "0",
```

Comment:
Are we disabling for some of these integration test suites because they're used by the`appsec_*` tests?

Reply:
Yes, this suite has appsec tests in it, but I understand that it is shared with the Django APM contrib.

Disabling ITR would have the effect of running all the tests every time (of this test suite), so no test skipping.